### PR TITLE
[FLOC-2464] Release Flocker 1.0.1pre1

### PR DESCRIPTION
**Pull request by wallrj**
Release branch for: https://clusterhq.atlassian.net/browse/FLOC-2464

DO NOT MERGE!!!

Ready for pre-tag review: http://doc-dev.clusterhq.com/gettinginvolved/infrastructure/release-process.html#pre-tag-review-process

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,5 +1,14 @@
-Flocker 1.0.0pre1 (2015-06-12)
-==============================
```
Comment (wallrj) [inline NEWS]: Github display issue? Master has `Flocker 1.0.0 (2015-06-15)` as expected.

File: README.rst [hunk 2]
```diff
@@ -45,7 +45,7 @@ Flocker is also tested using `continuous integration`_.
 .. _ClusterHQ: https://clusterhq.com/
 .. _Twisted: https://twistedmatrix.com
 .. _installing Flocker: https://docs.clusterhq.com/en/latest/indepth/installation.html
-.. _tutorial: https://docs.clusterhq.com/en/latest/gettingstarted/index.html
```
Comment (wallrj) [inline README.rst]: Another Github display issue. Master has `using/tutorial`.

**Comment by adamtheturtle** [comment]
With a previous release branch there were no git conflicts but GitHub showed the same "We can’t automatically merge this pull request." as is shown here, so that might be the case.

**Comment by adamtheturtle** [comment]
Itamar showed me how to see a more representative diff (https://github.com/ClusterHQ/flocker/compare/release/flocker-1.0.0?xxx in that case, with the xxx to ensure you don't get the cached version).

**Comment by wallrj** [comment]
Ok. The branch is now minimal. Ready for review.

File: NEWS (hunk 1 above)
Comment (wallrj) [inline NEWS]: This has corrected its self after merging various content back into master.

File: README.rst (hunk 2 above)
Comment (wallrj) [inline README.rst]: This correct  now too.

**Comment by dwgebler** [comment]
This is fine (excepting the known buildbot / resource limit issues that are causing some builds to fail). Please proceed.

**Pull request merged**